tests/manifests/vm_with_access_credentials.yaml

```diff
@@ -51,9 +51,9 @@ spec:
             secret:
               secretName: test-access-credentials-secret
           propagationMethod:
-            qemuGuestAgent:
-              users:
-              - "fedora"
+            qemuGuestAgent: {}
+            #  users:
```

Comment:
Is this a typo or intended?

Reply:
left that there to indicate the actual user used but I'm not sure anyone is looking at these yamls